### DISCUSSION TITLE
Android添加服务器后重启软件服务器空了

### DISCUSSION THREAD

**Discussion by lingkai0123** [lollipopkit/flutter_server_box#373] [General]
使用的是红米K40 系统是澎湃
![Screenshot_2024-06-05-17-01-22-856_tech.lolli.toolbox.jpg](https://github.com/lollipopkit/flutter_server_box/assets/63649357/9ec5b0af-99aa-48a9-9745-dbe0c1d15a42)
重启后：
![Screenshot_2024-06-05-17-02-08-545_tech.lolli.toolbox.jpg](https://github.com/lollipopkit/flutter_server_box/assets/63649357/97d6cced-157e-43c3-8898-d503eff3df07)

**Comment by lollipopkit** [comment]
仅添加服务器基础信息无法复现，是添加了什么额外设置吗？

**Reply by lingkai0123** [reply]
没有其他设置，都是安装默认的，换过其他版本也一样，不是每次重启都清空，还有时候重启删除一个，可能是我的手机不兼容吧(-̩̩̩-̩̩̩-̩̩̩-̩̩̩-̩̩̩___-̩̩̩-̩̩̩-̩̩̩-̩̩̩-̩̩̩)

**Reply by lollipopkit** [reply]
兼容性问题仅占 0.01 可能性。
开了webdav同步？或者保存失败（根本没有保存上），所以重启才会消失

**Reply by lingkai0123** [reply]
没开同步

**Reply by lingkai0123** [reply]
我下载了windows端的也是一样0.0

**Reply by lingkai0123** [reply]
windows端的第二次添加后重启几次都没事

**Reply by lollipopkit** [reply]
在 Android 上添加后，进入首页右上角的截图，重启，如果消失，发截图

**Reply by lingkai0123** [reply]
添加后有警告
![Screenshot_2024-06-05-18-11-30-637_tech.lolli.toolbox.jpg](https://github.com/lollipopkit/flutter_server_box/assets/63649357/3cfbb5ff-8275-4316-8ca6-ff57de98f952)

**Reply by lollipopkit** [reply]
wol 就是特殊设置。。。找到问题了

https://cdn.lolli.tech/serverbox/ServerBox_932_arm64.apk

**Reply by lingkai0123** [reply]
可以了，非常感谢
